packages/babel-generator/package.json

```diff
@@ -15,6 +15,7 @@
     "babel-runtime": "^6.9.0",
     "babel-types": "^6.14.0",
     "detect-indent": "^3.0.1",
+    "jsesc": "^1.3.0",
```

Comment:
AFAIK it looks to me that 1.x supports versions < 4 and 2.0 only supports 4+
https://github.com/mathiasbynens/jsesc/commit/e212a67946cd2a31578fac650f31281e52e4ca0c
But haven't tested.